Mathlib/Tactic/Linter/TextBased.lean

```diff
@@ -443,7 +443,7 @@ def lintModules (moduleNames : Array String) (mode : OutputSetting) : IO UInt32
   | OutputSetting.print style =>
     formatErrors allUnexpectedErrors style
     if numberErrorFiles > 0 && mode matches OutputSetting.print _ then
```

Comment:
```suggestion
    if allUnexpectedErrors.size > 0 && mode matches OutputSetting.print _ then
```
Should this then also be `allUnexpectedErrors.size`?

Reply:
This would be equally fine, but I'd say it doesn't matter.
`numberErrorFiles` is the *number* of files with an error, **allUnexpectedErrors.size** is the number of overall errors.

Do you think using the latter is clearer?

Reply:
Ok, I understand now: leaving what you initially had is fine, thanks!